runtime/include/komp.h

```diff
@@ -174,7 +174,7 @@ typedef void ident_t;
 
 typedef int kmp_int32;
 typedef long long kmp_int64;
-#if defined(TARGET_X8632) || defined(TARGET_LINUX_ARM32)
+#if __WORDSIZE == 32
```

Comment:
`__WORDSIZE` is not defined by default in MSVC (https://godbolt.org/z/M8rfsPhYn). A brief Google search shows that it is recommended to do this in portable software:
```
#ifdef _MSC_VER
#ifdef _WIN64
#define __WORDSIZE 64
#else
#define __WORDSIZE 32
#endif
#endif
```

Reply:
Comment @xoviat ?

Reply:
@bryanpkc I've added the snippet that you'd suggested. Is it portable enough now?